views-vues/assurance-level-requirement/resultTable.css

```diff
@@ -0,0 +1,39 @@
+.table-results {
```

Comment:
I'd pull this up to the assets folder

Reply:
Simplified by using more WET styles. 
resultTable.css has been deleted now.